src/pointcloud_to_laserscan_nodelet.cpp

```diff
@@ -234,46 +234,46 @@ namespace pointcloud_to_laserscan
 
       if (std::isnan(*iter_x) || std::isnan(*iter_y) || std::isnan(*iter_z))
       {
-        NODELET_DEBUG("rejected for nan in point(%f, %f, %f)\n", *iter_x, *iter_y, *iter_z);
+        NODELET_DEBUG("rejected for nan in point(%f, %f, %f)\n", static_cast<double>(*iter_x), static_cast<double>(*iter_y), static_cast<double>(*iter_z));
```

Comment:
Not for this PR, but curious about tx2 speed dealing with single or even half precision or fixed point. Once we get past floor filter, don’t need much precision at all.

Reply:
we should probably just run everything as float... unfortunately we have many places in codebase where we switch back and forth. 

my guess is that float is 2x as fast as double...